mkosi.profiles/desktop/mkosi.conf.d/debian/mkosi.conf

```diff
@@ -4,7 +4,6 @@
 Distribution=debian
 
 [Content]
-Splash=/usr/share/pixmaps/debian-logo.png
```

Comment:
might be cool of mkosi would try to convert this to the right form on its own btw

Reply:
More of a ukify thing, but yeah it's on the todo list